misc/emacs/dbscheme-mode.el

```diff
@@ -0,0 +1,27 @@
+
+(define-generic-mode
+    'dbscheme-mode ; mode name
+
+  ;; comments
+  '(("//" . nil)
+    ("/*" . "*/"))
+
+  ;; keywords
+  '("case" "ref" "unique" "of")
+
+  ;; other things to highlight
+  `((,ql/primitive-type-regex . 'font-lock-type-face)
+    ("\\<varchar([0-9]+)" . 'font-lock-type-face)
+    (,ql/at-type-regex 0 'font-lock-type-face)
+    (,ql/predicate-regex 1 'font-lock-variable-name-face))
+
+  ;; auto mode alist
+  '("\\.dbscheme?$")
```

Comment:
The `?` should not be there. Do we have `foo.dbshcem` files?

Reply:
Right, I think I must have copy-pasted from `.qll?` a long time ago!